index.js

```diff
@@ -85,6 +85,19 @@ Mailgen.prototype.generatePlaintext = function (params) {
     // Render the plaintext theme with ejs, injecting the data accordingly
     var output = ejs.render(this.cachedPlaintextTheme, ejsParams);
 
+    // Definition of the <br /> tag as a regex pattern
```

Comment:
I `git rebase -i HEAD~2` and `git push origin +master`so the last commit should be gone.